source/stylesheets/_tab-stack.scss.erb

```diff
@@ -192,11 +192,15 @@
 							display: flex;
```

Comment:
This is another issue I just noticed. If the machine types wrap into multiple lines, there are no distance between the rows:

<img width="826" alt="image" src="https://user-images.githubusercontent.com/97228168/169914268-8a38f9b7-b179-432c-9b2a-d7238317d053.png">

I think we could quickly fix this by:
```
.machine-types {
  ...
  gap: 18px;
}
```

and `margin-right: 18px` should be removed from `.machine-type-wrapper` at the same time.